content/docs/2.0/concepts/scaling-jobs.md

```diff
@@ -49,24 +50,27 @@ You can find all supported triggers [here](../scalers).
 
 ```yaml
   jobTargetRef:
-    parallelism: 1 # Max number of desired instances ([docs](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#controlling-parallelism))
-    completions: 1 # Desired number of successfully finished instances ([docs](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#controlling-parallelism))
-    activeDeadlineSeconds: 600 # Specifies the duration in seconds relative to the startTime that the job may be active before the system tries to terminate it; value must be positive integer
-    backoffLimit: 6 # Specifies the number of retries before marking this job failed. Defaults to 6
+    parallelism: 1              # Max number of desired instances ([docs](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#controlling-parallelism))
+    completions: 1              # Desired number of successfully finished instances ([docs](https://kubernetes.io/docs/concepts/workloads/controllers/jobs-run-to-completion/#controlling-parallelism))
+    activeDeadlineSeconds: 600  # Specifies the duration in seconds relative to the startTime that the job may be active before the system tries to terminate it; value must be positive integer
+    backoffLimit: 6             # Specifies the number of retries before marking this job failed. Defaults to 6
 ```
 
 The optional configuration parameter. Currently not implemented. It is going to be supported. 
```

Comment:
@TsuyoshiUshio I don't understand this note? Is this some leftover that could be removed? Thanks

Reply:
Sorry, Probably, It was my mistake remove it. :) I double checked if the PollingInterval is valid or not. It is valid.